pkg/kubectl/rollback.go

```diff
@@ -269,11 +274,15 @@ func (r *DaemonSetRollbacker) Rollback(obj runtime.Object, updatedAnnotations ma
 	return rollbackSuccess, nil
 }
 
+// StatefulSetRollbacker implements the Rollbacker interface.
 type StatefulSetRollbacker struct {
 	c clientset.Interface
 }
 
-// toRevision is a non-negative integer, with 0 being reserved to indicate rolling back to previous configuration
+// toRevision
```

Comment:
nit: you've got an incomplete comment here.

Reply:
thanks for the review @ericchiang, will fix this up shortly.